schwab/streaming.py

```diff
@@ -1041,113 +1042,128 @@ class LevelOneFuturesFields(_BaseFieldEnum):
         streaming-data#_Toc504640603>`__
         '''
 
-        #: UNKNOWN
-        FIELD_0 = 0
+        #: Ticker symbol in upper case.
+        SYMBOL = 0
 
-        #: UNKNOWN
-        FIELD_1 = 1
+        #: Current Best Bid Price
+        BID_PRICE = 1
 
-        #: UNKNOWN
-        FIELD_2 = 2
+        #: Current Best Ask Price
+        ASK_PRICE = 2
 
-        #: UNKNOWN
-        FIELD_3 = 3
+        #: Price at which the last trade was matched
+        LAST_PRICE = 3
 
-        #: UNKNOWN
-        FIELD_4 = 4
+        #: Number of contracts for bid
+        BID_SIZE = 4
 
-        #: UNKNOWN
-        FIELD_5 = 5
+        #: Number of contracts for ask
+        ASK_SIZE = 5
 
-        #: UNKNOWN
-        FIELD_6 = 6
+        #: Exchange with the best bid
+        BID_ID = 6
 
-        #: UNKNOWN
-        FIELD_7 = 7
+        #: Exchange with the best ask
+        ASK_ID = 7
 
-        #: UNKNOWN
-        FIELD_8 = 8
+        #: Aggregated contracts traded throughout the day, including pre/post market hours.
+        TOTAL_VOLUME = 8
 
-        #: UNKNOWN
-        FIELD_9 = 9
+        #: Number of contracts traded with last trade
+        LAST_SIZE = 9
 
-        #: UNKNOWN
-        FIELD_10 = 10
+        #: Time of the last quote in milliseconds since epoch
+        QUOTE_TIME = 10
 
-        #: UNKNOWN
-        FIELD_11 = 11
+        #: Time of the last trade in milliseconds since epoch
+        TRADE_TIME = 11
```

Comment:
Please follow the `TIME_MILLIS` convention used through this file.

Reply:
Understood. Done.